docs/src/main/sphinx/sql/show-catalogs.rst

```diff
@@ -12,5 +12,10 @@ Synopsis
 Description
 -----------
 
-List the available catalogs.
-The ``LIKE`` clause can be used to restrict the list of catalog names.
+List the available catalogs. 
+
+:ref:`Specify a pattern <like_operator>` in the optional `LIKE` clause to filter
+the results to the desired subset. For example, the following query allows you
+to find catalogs where the first character in a value is specified::
```

Comment:
"that start with the `t` character::"

Reply:
Add this at the end of the last sentence to detail what the example actually does..  and same for others below